generated/gitops-template/jenkins/Jenkinsfile

```diff
@@ -12,6 +12,13 @@ pipeline {
         COSIGN_SECRET_KEY = 'dummy'
         /* Used to verify the image signature and attestation */
         COSIGN_PUBLIC_KEY = credentials('COSIGN_PUBLIC_KEY')
+        /* URL of the BOMbastic api host (e.g. https://sbom.trustification.dev) */
+        TRUSTIFICATION_BOMBASTIC_API_URL = credentials('TRUSTIFICATION_BOMBASTIC_API_URL')
+        /* URL of the OIDC token issuer (e.g. https://sso.trustification.dev/realms/chicken) */
+        TRUSTIFICATION_OIDC_ISSUER_URL = credentials('TRUSTIFICATION_OIDC_ISSUER_URL')
+        TRUSTIFICATION_OIDC_CLIENT_ID = credentials('TRUSTIFICATION_OIDC_CLIENT_ID')
+        TRUSTIFICATION_OIDC_CLIENT_SECRET = credentials('TRUSTIFICATION_OIDC_CLIENT_SECRET')
+        TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION = credentials('TRUSTIFICATION_SUPPORTED_CYCLONEDX_VERSION')
```

Comment:
Only one of these is an actual secret (TRUSTIFICATION_OIDC_CLIENT_SECRET), but I don't know if there's any better way to set this configuration :thinking:

Reply:
> Only one of these is an actual secret (TRUSTIFICATION_OIDC_CLIENT_SECRET), but I don't know if there's any better way to set this configuration 🤔

The rhtap/env.sh holds the per-repo environment variables which are loaded by the pipeline scripts 
These env vars were originally defined as the values for the pipeline parameters and was the portable way to allow user defined env vars. Each CI has its own mechanism so we can consider evolving pipelines to use those.

Reply:
The thing is that all these values are properties of a specific RHTAP deployment, they're not something that a user should have to set. IMO they should ideally be set by the platform engineer

Reply:
In that sense, credentials might be the right choice in Jenkins (the unfortunate part is that defining them as credentials means Jenkins will censor them in the logs, so the URLs are hidden)

Reply:
> The thing is that all these values are properties of a specific RHTAP deployment, they're not something that a user should have to set. IMO they should ideally be set by the platform engineer

I agree. Yes, the site install defaults are set by the platform engineer when they fork the templates repo so in theory they fork the templates and update the defaults, Currently the can set the URLs for site things like quay url, glab url, org default and could do the same for env. Its not "clean" and I would prefer the templates don't need to be customized (so they are easier to update) and the values get installed into RHDH as some sort of site profile which could have multiples (per team etc) and reduce the duplication per repo. For not as the secrets across CIs can be "org" or global based and shared in multiple repos this is good enough